extension_templates/forecasting.py

```diff
@@ -215,42 +212,46 @@ def _update(self, y, X=None, update_params=True):
 
     # todo: consider implementing this, optional
     # if not implementing, delete the method
-    def _update_predict_single(
-        self,
-        y,
-        fh,
-        X=None,
-        update_params=True,
-        return_pred_int=False,
-        alpha=DEFAULT_ALPHA,
-    ):
+    def _update_predict_single(self, y, fh, X=None, update_params=True):
         """Update forecaster and then make forecasts.
 
         Implements default behaviour of calling update and predict
         sequentially, but can be overwritten by subclasses
         to implement more efficient updating algorithms when available.
         """
         self.update(y, X, update_params=update_params)
-        return self.predict(fh, X, return_pred_int=return_pred_int, alpha=alpha)
+        return self.predict(fh, X)
 
         # implement here
         # IMPORTANT: avoid side effects to y, X, fh
 
-    # todo: consider implementing this, optional
-    # if not implementing, delete the method
-    def _compute_pred_int(self, alphas):
-        """Calculate the prediction errors for each point.
+    def _predict_quantiles(self, fh, X=None, alpha=DEFAULT_ALPHA):
+        """
+        Compute/return prediction quantiles for a forecast.
+
+        Must be run *after* the forecaster has been fitted.
+
+        If alpha is iterable, multiple quantiles will be calculated.
+
+        Users can implement _predict_interval if calling it makes this faster.
 
         Parameters
         ----------
-        alpha : float or list, optional (default=0.95)
-            A significance level or list of significance levels.
+        fh : int, list, np.array or ForecastingHorizon
+            Forecasting horizon, default = y.index (in-sample forecast)
```

Comment:
there is no `y` here, so the default does not seem to make sense - or does it and I'm missing sth?
If this is a typo, please remove the "default" part.